nemo/collections/nlp/models/language_modeling/megatron_lm_encoder_decoder_model.py

```diff
@@ -1259,6 +1265,107 @@ def dummy():
         )
         return predicted_tokens_dec, log_probs
 
+    def log_prob(
```

Comment:
Very useful method! Would it make sense to call `log_prob` within `decode`? They seem to have a lot of overlapping code regarding the logits computation